lib/src/source/hosted.dart

```diff
@@ -460,6 +469,7 @@ class BoundHostedSource extends CachedSource {
     if (!dirExists(cacheDir)) return [];
 
     return listDir(cacheDir)
+        .where(_looksLikePackageDir)
```

Comment:
Again, we don't need `_looksLikePackageDir`, I don't know if we should remove it.. But that is correct right?